ecosystem/sep-sign-and-verify-messages.md

```diff
@@ -0,0 +1,174 @@
+## Preamble
+
+```
+SEP: <TBD>
+Title: Sign and Verify Messages
+Author: Jun Luo (@overcat), Pamphile Roy (@tupui), Tomer Weller(@tomerweller), Leigh McCulloch (@leighmcculloch), OrbitLens (@orbitlens), Enrique Arrieta (@earrietadev), Piyal Basu (@piyalbasu), MonsieurNicolas (@MonsieurNicolas)
```

Comment:
The author list looks very long, I can definitely be removed. It just needs to list the actual author or anyone you feel like provided substantial contribution to be considered an author, not folks who were consulted or participated in the GitHub Discussion and just provided feedback. The discussion link below provides a good history.

```suggestion
Author: Jun Luo (@overcat), Pamphile Roy (@tupui), Tomer Weller(@tomerweller), OrbitLens (@orbitlens), Enrique Arrieta (@earrietadev), Piyal Basu (@piyalbasu), MonsieurNicolas (@MonsieurNicolas)
```

Reply:
Indeed, I removed Enrique Arrieta as well because a long author list would cause significant changes in the git history of the readme file.